Sources/FontInspector.swift

```diff
@@ -177,6 +177,14 @@
         }
 
     }
+    
+    extension Fractions: MappableFeature {
+        
```

Comment:
#req Whitespace-only lines should have their trailing whitespace trimmed. This one is Xcode's fault - it fails to correctly strip the blank lines when pasting a block like this 😞